source/beamline/CLS/CLSBeamlineStatus.cpp

```diff
@@ -33,11 +38,7 @@ bool CLSBeamlineStatus::isOn() const
 
 bool CLSBeamlineStatus::isOff() const
 {
-	if (beamlineStatusPVControl_) {
-		return (!isChildState1(beamlineStatusPVControl_));
-	} else {
-		return (isConnected() && !areAllChildrenState1());
-	}
+	return isConnected() && !isOn();
 }
 
 AMControl * CLSBeamlineStatus::beamlineStatusPVControl()
```

Comment:
I don't think BioXAS does, and I'm just curious--do many beamlines have a beamline status PV control like this?